The default filter search path is /usr/libexec/cups/filter. MacPorts can't
write to this location, so specify an absolute path instead.

--- brlaser.drv.in.orig
+++ brlaser.drv.in
@@ -29,7 +29,7 @@ Manufacturer "Brother"
 Version "@BRLASER_VERSION@"
 
 // Each filter provided by the driver...
-Filter application/vnd.cups-raster 33 rastertobrlaser
+Filter application/vnd.cups-raster 33 @CUPS_SERVER_BIN@/filter/rastertobrlaser
 
 // Supported resolutions.
 // The 1200dpi mode is weird: we need to send 1200x1200dpi raster
